01_01_Basic_Rules.txt

```diff
@@ -628,6 +626,33 @@ Class Feats:
 specifies a class feat. Characters can not take non-class feats instead of class feats 
 or vice-versa. Class Feats are listed in the Classes document.
 
+==============================
+Items and Buying Items
+==============================
+
+	When you build a character, you will get some starting money and the time to
+spend it to acrue the kit that you'll start playing with. But this likely won't
+be the last time that you'll have the opportunity to buy things. It may be smart
+to save some of your starting cash to spend elsewhere. Some stores and vendors
+you meet will sell exactly what you see in the Player Handbook. Some traders 
+may offer items or weapons that aren't available either at the start of the game
+or anywhere else in the galaxy! 
+
+	When you buy a gun, the gun usually comes with 5 magazines of ammunition
+unless otherwise stated. If you spend an Action to pick up a weapon from a 
```

Comment:
I would leave out the 'spend an Action'. It will almost always cost an Action to pick up an item, but if it did eventually change, we'd change it in the action rules and never remember to change it here too. DRY it out please 😃

Reply:
oh snap! With the right acronym and everything! You're absolutely right!